server/src/main/java/org/elasticsearch/search/aggregations/support/package-info.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * The Aggregations Support package holds shared code for the aggregations framework, especially around dealing with values.
+ *
+ * Key Classes
+ *
+ * {@link org.elasticsearch.search.aggregations.support.ValuesSource} and its subclasses
```

Comment:
Since this is in the package do you need to fully qualify it? In general I'm fine importing things *only* for Javadoc if it makes the javadoc easier to read.